src/GafferUIBindings/PlugAdderBinding.cpp

```diff
@@ -89,13 +92,71 @@ struct PlugMenuSlotCaller
 
 };
 
+struct PlugAdderWrapper : public PlugAdder, IECorePython::Wrapper< PlugAdder >
+{
+
+	PlugAdderWrapper(PyObject *self, StandardNodeGadget::Edge edge ) : PlugAdder( edge ), IECorePython::Wrapper<PlugAdder>( self, this )
+	{
+	}
+
+	bool acceptsPlug( const Gaffer::Plug *connectionEndPoint ) const
+	{
+		IECorePython::ScopedGILLock gilLock;
+		override o = this->get_override( "acceptsPlug" );
+		if( o )
+		{
+			try
+			{
+				return o( Gaffer::PlugPtr( const_cast<Gaffer::Plug *>( connectionEndPoint ) ) );
+			}
+			catch ( error_already_set )
+			{
+				PyErr_Print();
+			}
+		}
+		else
+		{
+			// TODO: replace this with proper logging?
+			printf( "PlugAdder: acceptsPlug() python method not defined" );
```

Comment:
Can we replace this with an Exception instead, as in the ExecutableRenderWrapper above?